apps/registry/scripts/jobs/vectorize.js

```
@@ -26,7 +30,7 @@ async function main() {
   console.log('Fetched jobs:', data);
 
   for (const job of data) {
-    if (!job.embedding_v5) {
+    if (true || !job.embedding_v5) {
```

Biome

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)